pkg/event/target/kafka.go

```diff
@@ -40,10 +40,11 @@ const (
 	KafkaTopic         = "topic"
 	KafkaQueueDir      = "queue_dir"
 	KafkaQueueLimit    = "queue_limit"
-	KafkaTLS           = "tls"
+	KafkaTLS           = "tls_enable"
 	KafkaTLSSkipVerify = "tls_skip_verify"
 	KafkaTLSClientAuth = "tls_client_auth"
-	KafkaSASL          = "sasl"
```

Comment:
Same goes for sasl_enable

Reply:
same:

in the readme you have provided that string: 
```
notify_kafka:1 tls_skip_verify="off"  queue_dir="" queue_limit="0" sasl_enable="off" sasl_password="" sasl_username="" tls_client_auth="0" tls_enable="off" brokers="" topic="" client_tls_cert="" client_tls_key=""
```
as you can see ` tls_enable="off"` and a ` sasl_enable="off" `

i just follow the readme file

Reply:
done in [c3e2365](https://github.com/minio/minio/pull/8593/commits/c3e23659564a6d7cc21b86f6fed033da8b2208cd)